src/lib.rs

```diff
@@ -322,6 +322,11 @@ pub enum Protocol {
     Tlsv11,
     /// The TLS 1.2 protocol.
     Tlsv12,
+    /// The TLS 1.3 protocol.
+    ///
+    /// Requires OpenSSL 1.1.1 or LibreSSL 3.4.0 or newer.
+    #[cfg(have_tls13_version)]
+    Tlsv13,
```

Comment:
I'd prefer to expose a uniform interface rather than having this be conditionally defined based on the backend version.

Reply:
I switched to runtime errors when unsupported.